widget/button.go

```diff
@@ -206,16 +188,15 @@ func (b *Button) CreateRenderer() fyne.WidgetRenderer {
 	objects := []fyne.CanvasObject{
 		text,
 	}
-	var shadow *shadow
-	if !b.HideShadow {
-		shadow = newShadow(shadowAround, theme.Padding()/2)
-		objects = append(objects, shadow)
+	shadowLevel := baseLevel
```

Comment:
Looking at the diff aren't baseLevel and buttonLevel in this snippet the wrong way round?